app/src/main/res/values/strings.xml

```diff
@@ -810,16 +810,16 @@
     <string name="security">Security</string>
     <string name="set_mode_for_app_ops_dots">Set mode for app ops…</string>
     <string name="backup_source_description">Back up the entire source directory.</string>
-    <string name="backup_apk_only_description">Back up only the APK files from the source directory. <b>Source</b> must be selected for this to work.</string>
-    <string name="backup_data_description">Back up data folders. If selected, internal data folders will be backed up by default.</string>
-    <string name="backup_external_data_description">Back up external data folders. <b>Data</b> must be selected for this to work.</string>
+    <string name="backup_apk_only_description">Only backs up APK files from the source directory.</string>
```

Comment:
The part about b>Source</b> needing to be selected is gone. Can't remember that being unimportant. Mistake to remove it?